ci/jenkinsfiles/release.groovy

```diff
@@ -264,6 +264,60 @@ pipeline {
         }
       }
     }
+    stage('Deploy ColdStorage Preview') {
```

Comment:
a question more for @doulba :) 

Do we have to copy past the same code. I mean one for the common build and a second for the release?

thx

Reply:
Hello @RSalem07 
Unfortunately having shared libs don't avoid to duplicate stage name

Reply:
commons libs file will be deleted by Pedro